app/src/main/java/com/example/android/receitaboa/activity/EditarReceitaActivity.java

```diff
@@ -204,6 +200,8 @@ protected void onActivityResult(int requestCode, int resultCode, @Nullable Inten
                 //Se a foto foi selecionada ou da camera ou da galeria, temos uma imagem
                 if (fotoReceita != null){
 
+                    abrirDialogCarregamento("Aguarde enquanto a foto é carregada!");
```

Comment:
Não se deve pôr uma string diretamente dentro do programa, extrair a string para um arquivo de tradução (string.xml)

https://www.andtuts.com/why-and-how-to-use-string-resources/